### DISCUSSION TITLE
superset_config.py is being ignored

### DISCUSSION THREAD

**Discussion by max-allan-surevine** [apache/superset#11408] [General]
I followed the install from scratch guide (without using virtualenv).

Started up the server with `superset run -h 0.0.0.0 -p8088` and it is fine. I can browse to superset and see the log messages in the terminal.

I then created a superset_config.py with the following content :
```
[ec2-user@ip-10-66-2-195 ~]$ cat /home/ec2-user/superset_config.py 
SUPERSET_WEBSERVER_PROTOCOL = "http"
SUPERSET_WEBSERVER_ADDRESS = "0.0.0.0"
SUPERSET_WEBSERVER_PORT = 8088
```

### Expected results

```
[ec2-user@ip-10-66-2-195 ~]$ superset run
Loaded your LOCAL configuration at [/home/ec2-user/superset_config.py]
logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/usr/local/lib64/python3.8/site-packages/flask_caching/__init__.py:191: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  warnings.warn(
No PIL installation found
INFO:superset.utils.screenshots:No PIL installation found
 * Serving Flask app "superset"
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
INFO:werkzeug: * Running on http://0.0.0.0:8088/ (Press CTRL+C to quit)
```

### Actual results

```
[ec2-user@ip-10-66-2-195 ~]$ superset run
Loaded your LOCAL configuration at [/home/ec2-user/superset_config.py]
logging was configured successfully
INFO:superset.utils.logging_configurator:logging was configured successfully
/usr/local/lib64/python3.8/site-packages/flask_caching/__init__.py:191: UserWarning: Flask-Caching: CACHE_TYPE is set to null, caching is effectively disabled.
  warnings.warn(
No PIL installation found
INFO:superset.utils.screenshots:No PIL installation found
 * Serving Flask app "superset"
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```

#### Screenshots

N/A

#### How to reproduce the bug

1. Follow instructions https://superset.apache.org/docs/installation/installing-superset-from-scratch
2. Create config file as above
3. Start superset

### Environment

(please complete the following information):

- superset version: `Superset 0.37.2`
- python version: `Python 3.8.5`
- node.js version: `-bash: node: command not found`
- npm version: `-bash: npm: command not found`

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [x ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x ] I have reproduced the issue with at least the latest released version of superset.
- [x ] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

It would be helpful if the above request for stacktraces told you where the logs are stored.

**Comment by betodealmeida** [comment]
Currently the environment variables `SUPERSET_WEBSERVER_*` are only used for the Celery cache, but it would be nice to make it consistent and use it for other things.
